src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -90,23 +90,20 @@ export default function GamePage(): JSX.Element | null {
   const isInstalling = status === 'installing'
   const isPlaying = status === 'playing'
   const isUpdating = status === 'updating'
+  const isQueued = status === 'queued'
   const isReparing = status === 'repairing'
   const isMoving = status === 'moving'
-  const hasDownloads = Boolean(
-    libraryStatus.filter(
-      (game) => game.status === 'installing' || game.status === 'updating'
-    ).length
-  )
+
+  const backRoute = location?.state?.fromDM ? '/download-manager' : '/'
```

Comment:
`location` can't be undefined/null. If `state` can be undefined/null, please mark it as such in the `as { state: ... }` above